src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/PostClaimOrchestrationHandler.java

```diff
@@ -106,16 +105,22 @@ public void citizenIssueHandler(CitizenClaimCreatedEvent event) {
             PDF sealedClaimPdf = documentOrchestrationService.getSealedClaimPdf(claim);
             PDF claimIssueReceiptPdf = documentOrchestrationService.getClaimIssueReceiptPdf(claim);
 
-            Supplier<Claim> updatedClaim = () ->
+            Claim updatedClaim =
                 doPinOperation
                     .andThen(c -> uploadSealedClaimOperation.perform(c, authorisation, sealedClaimPdf))
                     .andThen(c -> uploadClaimIssueReceiptOperation.perform(c, authorisation, claimIssueReceiptPdf))
                     .andThen(c -> rpaOperation.perform(c, authorisation, sealedClaimPdf))
                     .andThen(c -> notifyClaimantOperation.perform(c, event))
                     .apply(claim);
 
-            claimService.updateClaimState(authorisation, updatedClaim.get(), ClaimState.OPEN);
-
+            updatedClaim.getState()
+                .filter(state -> state == CREATE)
+                .ifPresent(state -> {
+                    claimService.updateClaimState(
+                        authorisation,
+                        updatedClaim,
+                        state);
+                });
```

Comment:
`.filter(state -> state == CREATE)` could be simplified to `.filter(Predicate.isEqual(CREATE))`
(static import on `isEqual` to further simplify)
`.ifPresent(...)` body doesn't need braces as it's a single statement.

Reply:
done.